src/Form/atoms/FilterOption.tsx

```diff
@@ -170,7 +173,7 @@ const FilterOption: React.FC<IFilterOption> = ({
         </FakeCheckbox>
       )}
       {(optionType === 'radio') && <FakeRadioButton><FakeInnerRadio /></FakeRadioButton>}
-      <Title>{title}</Title>
+      <Title title={title}>{title}</Title>
```

Comment:
In general showing title in FilterDropdown is not a good idea, but if any reason we need to show it then it should be conditional and should only shown for the options which are doesn't fit in the dropdown width.

**pointer-events: none;** and **title** should be conditional.

Do not show **title** for the FilterOption which are fit perfectly in dropdown.

Some code reference are available here
https://codepen.io/triple-j/pen/wadKQB
https://dirask.com/posts/JavaScript-check-if-text-overflow-ellipsis-is-active-1XB5Yp